garrysmod/gamemodes/terrortown/gamemode/player.lua

```diff
@@ -546,9 +546,9 @@ local function CheckCreditAward(victim, attacker)
       end
 
       local pct = inno_dead / inno_total
-      if pct >= GetConVarNumber("ttt_credits_award_pct") then
+      if pct >= GetConVar("ttt_credits_award_pct"):GetInt() then

```

Comment:
This'll screw up even with the default setting of `ttt_credits_award_pct 0.35` because GetInt returns a whole number.